app/main.py

```diff
@@ -1,3 +1,55 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[float, int, str]) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: Union[float, int, str]) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        return self
+
+    def __mul__(self, scalar: float) -> "Distance":
+        return Distance(self.km * scalar)
+
+    def __truediv__(self, divisor: float) -> "Distance":
+        return Distance(round(self.km / divisor, 2))
+
+    def __lt__(self, other: Union[float, int, str]) -> bool:
```

Comment:
it doesn't accept str, it should accept only float, int and Distance
fix it everywhere